apps/src/code-studio/pd/workshop_enrollment/workshop_enroll.jsx

```diff
@@ -23,6 +23,7 @@ const SUBMISSION_STATUSES = {
 
 export default class WorkshopEnroll extends React.Component {
   static propTypes = {
+    user_id: PropTypes.number.isRequired,
```

Comment:
where is this prop coming from? said another way, does a different component need to be updated to pass this through?

Reply:
This is passed in from the `workshop_enrollment_controller` here: https://github.com/code-dot-org/code-dot-org/pull/54826/files#diff-658aa36aa0f026bca4b58f798c1e3bdf2fa93625441c01de0f588dbcdc6a678aR74

Reply:
oof that's confusing 😅 but definitely out of the scope of this PR to fix so LGTM!

Reply:
Yes totally agree! Sounds good thank you!